examples/curl.rs

```diff
@@ -104,18 +104,30 @@ impl http::Merge for ModuleConfig {
 }
 
 http_request_handler!(curl_access_handler, |request: &mut http::Request| {
+    // we can check if a request is internal and disable handler
+    let log = request.log();
+    ngx_log_debug!(log, "is internal: {}", request.is_internal());
```

Comment:
When this is printed will there be enough context for a user to know what's internal? Is there a convention around module identification for context?